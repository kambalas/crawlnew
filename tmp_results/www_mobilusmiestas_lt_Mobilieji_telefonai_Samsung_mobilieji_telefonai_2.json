[
  {
    "index": 3,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "litellm.APIError: APIError: DeepseekException - peer closed connection without sending complete message body (incomplete chunked read)"
  },
  {
    "name": "Samsung Galaxy S24 FE 128GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Plus 512GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 256GB 12RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 512GB 12RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 128GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 256GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 256GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 512GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 512GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 256GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy Xcover 7 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy A05s 128GB 4GB RAM LTE",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy A16 128GB 4GB RAM LTE",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy A16 5G 128GB 4GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A26 128GB 6GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A26 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A36 128GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A36 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A55 128GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A55 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A56 128GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A56 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy M35 128GB 6GB RAM 5G",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy M55 128GB 8GB RAM 5G",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY S23 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 128GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 256GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 FE 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 FE 128GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "Samsung Galaxy S25 Ultra 256GB 12GB RAM 5G Dual Sim",
    "price": "1025.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 128GB 5G Dual Sim",
    "price": "538.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy M55 128GB 8GB RAM 5G",
    "price": "277.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 512GB 12GB RAM 5G Dual Sim",
    "price": "1159.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 256GB 12GB RAM 5G Dual Sim",
    "price": "748.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 128GB 12GB RAM 5G Dual Sim",
    "price": "674.00",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A56 256GB 8GB RAM",
    "price": "372.00",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A36 256GB 8GB RAM",
    "price": "314.00",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A26 128GB 6GB RAM",
    "price": "239.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy Xcover 7 128GB",
    "price": "246.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 256GB 5G Dual Sim",
    "price": "875.00",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 512GB 5G Dual Sim",
    "price": "959.00",
    "error": false
  }
]