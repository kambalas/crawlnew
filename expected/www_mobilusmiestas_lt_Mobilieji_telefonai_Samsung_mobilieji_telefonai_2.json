[
  {
    "name": "SAMSUNG GALAXY A56 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy M35 128GB 6GB RAM 5G",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy M55 128GB 8GB RAM 5G",
    "price": "",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY S23 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 128GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 256GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 FE 256GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 FE 128GB 8GB RAM",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Plus 512GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 256GB 12RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 Ultra 512GB 12RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 128GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 256GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 256GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 512GB 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 512GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 256GB 12GB RAM 5G Dual Sim",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy Xcover 7 128GB",
    "price": "",
    "error": false
  },
  {
    "index": 3,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  },
  {
    "name": "Apple iPhone 15 Pro Max dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 Pro dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 Pro Max dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 mini dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12/12 Pro dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 Pro Max dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 Mini dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 11 Pro Max dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 11 Pro dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 11 dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone XR dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone Xs Max dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone X/Xs dėklai",
    "price": "",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 256GB 12RAM 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 512GB 12GB RAM 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 128GB 12GB RAM 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 256GB 12GB RAM 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 256GB 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 512GB 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 512GB 12GB RAM 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 256GB 12GB RAM 5G Dual Sim",
    "price": "0",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 256GB 12GB RAM 5G Dual-Sim",
    "price": "1025",
    "error": false
  },
  {
    "name": "Samsung Galaxy S24 128GB 5G Dual-Sim",
    "price": "538",
    "error": false
  },
  {
    "name": "Samsung Galaxy M55 128GB 8GB RAM 5G",
    "price": "277",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Ultra 512GB 12GB RAM 5G Dual-Sim",
    "price": "1159",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 256GB 12GB RAM 5G Dual-Sim",
    "price": "748",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 128GB 12GB RAM 5G Dual-Sim",
    "price": "674",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A56 256GB 8GB RAM",
    "price": "372",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A36 256GB 8GB RAM",
    "price": "314",
    "error": false
  },
  {
    "name": "SAMSUNG GALAXY A26 128GB 6GB RAM",
    "price": "239",
    "error": false
  },
  {
    "name": "Samsung Galaxy Xcover 7 128GB",
    "price": "246",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 256GB 5G Dual-Sim",
    "price": "875",
    "error": false
  },
  {
    "name": "Samsung Galaxy S25 Plus 512GB 5G Dual-Sim",
    "price": "959",
    "error": false
  }
]